formSchemas.js

```diff
@@ -1,5 +1,35 @@
 const API = require('./api')
 
+/**
+ * Runs an array of validators over a value
+ * this is a workaround I found to allow multiple custom validators
+ * but still maintain individual error messages
+ */
+const validationArray = validators => {
+	let errors = [];
+
+	return {
+		custom: {
+			errorMessage: () => {
+				return errors.length ? errors[0] : 'value is invalid';
+			},
+			options: (value, { req }, opts) => {
+				errors = [];
+				const results = validators.map((validator, index) => {
+					const result = validator.validate(value, req, opts);
+					// If validation failed set current Error
+					if (result === false) {
+						const errorMessage = validator.errorMessage || `${value} is invalid`;
+						errors.push(errorMessage);
+					}
+					return result;
+				});
+				return results.every(value => value === true);
+			}
+		}
+	};
+};
```

Comment:
I think your editor likes 4 spaces and mine likes 2. Can we standardize on 2? 

I'm going to push another commit on here.

Reply:
I'm using VSCode. Are you using it as well or a different one?

Reply:
using it the same, I keep trying to keep 2 spaces, but it protests! will clean it up though